src/main/java/net/greatstart/model/User.java

```diff
@@ -1,7 +1,10 @@
 package net.greatstart.model;
 
+import lombok.Data;
+
 import java.util.List;
 
+@Data
 public class User {
     private int id;
     private String name;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "name" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)